CHANGELOG.md

```diff
@@ -15,6 +15,18 @@ New Features
   December 2018 and now was by overriding the default SDP format to Plan B. Starting with this version,
   twilio-video.js will use Unified Plan where available, while also maintaining support for earlier
   browser versions with Plan B as the default SDP format. (JSDK-2312)
+  
+  **NOTE:**
+  
+  Typically, the size of a Unified Plan SDP is greater than a Plan B SDP. This will result in some
```

Comment:
Suggest instead of 

> Typically, the size of a Unified Plan SDP is greater than a Plan B SDP. This will result in some increased signaling traffic whenever Participants join/leave a Room or publish/unpublish Tracks. Our load tests...

Change to:
> With Unified Plan, SDPs generated are larger than that of Plan B, it leads to an increase in signaling playloads. Our load tests...

Reply:
My 2 cents:
With Unified Plan, SDPs generated are larger than that of Plan B. It leads to an increase in signaling traffic at the time of negotiation - when participants join/leave a room or publish/unpublish tracks. 
Our load tests with group rooms of 35+ participants with media tracks revealed up to 160% overall increase in signaling traffic with Unified plan vs Plan B. The tests did not show any change in the media traffic. We also noticed up to 20% increase in CPU usage in 35+ participants group rooms, which ...